src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] resultingArray = new int[3];
+        try (BufferedReader readFile = Files.newBufferedReader(Paths.get(fromFileName))) {
+            String line;
+
+            while ((line = readFile.readLine()) != null) {
+                String[] tokens = line.split(",");
+                if (tokens[0].equals("supply")) {
```

Comment:
тут ще можна для інформативності індекси в константи повиносити 
```suggestion
                if (tokens[OPERATION_INDEX].equals("supply")) {
```

Reply:
я спочатку так і зробив, але для компактності змінив на список і все
хороша практика це оголошення таких змінних?